libjsengine/JSEngine.cpp

```diff
@@ -0,0 +1,36 @@
+/*
+	This file is part of cpp-ethereum.
+
+	cpp-ethereum is free software: you can redistribute it and/or modify
+	it under the terms of the GNU General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	cpp-ethereum is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+	GNU General Public License for more details.
+
+	You should have received a copy of the GNU General Public License
+	along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
+*/
+/** @file JSEngine.cpp
+ * @author Marek Kotewicz <marek@ethdev.com>
+ * @date 2015
+ * Ethereum client.
+ */
+
+#include <string.h>
+#include <stdlib.h>
+#include "JSEngine.h"
+
+using namespace dev;
+using namespace dev::eth;
+
+JSString::JSString(char const* _cstr): m_cstr(strdup(_cstr)) {}
```

Comment:
mac version of v8 (from homebrew) is built with old `libstdc++`. There are few [ABI differences](http://stackoverflow.com/questions/17450130/what-is-a-good-way-of-handling-abi-differences-between-libc-and-the-older-libs/17454642#17454642) between `libc++` and `libstdc++`.